packages/oid4vci-demo-frontend/src/configs/legal-entity.banqup.json

```diff
@@ -234,7 +261,12 @@
         {
           "id": "issueSuccess",
           "operation": "navigate",
-          "path": "https://banqup.demo.sphereon.com"
+          "path": "/credentials/issue/success"
+        },
+        {
+          "id": "toCompanyFlow",
```

Comment:
does not go to the company flow. this goes to the account flow

Reply:
changed it to `toAccountFlow`